scripts/copy-files.ts

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require('fs')
```

Comment:
Using `esno` we could use `import` here

Reply:
we need to add `esno` to `dev dependencies`, you have it globally...

Reply:
```shell
F:\work\projects\quini\GitHub\antfu\vite-ssg>pnpm run example:dev

> vite-ssg@0.11.2 example:dev F:\work\projects\quini\GitHub\antfu\vite-ssg
> npm run copy-files && npm -C examples/multiple-pages run dev


> vite-ssg@0.11.2 copy-files
> esno scripts/copy-files.ts

"esno" no se reconoce como un comando interno o externo,
programa o archivo por lotes ejecutable.
 ERROR  Command failed with exit code 1.
```